libs/journey-bundles/transactions/src/lib/transactions.bundle.ts

```diff
@@ -44,38 +45,32 @@ const routes: Routes = transactionsJourney(
   })
 );
 
-@NgModule({
-  imports: [RouterModule.forChild(routes)],
-  providers: [
-    TransactionsConfigService,
-    TransactionsRouteTitleResolverService,
-    {
-      provide: TransactionsJourneyConfiguration,
-      useFactory: (configService: TransactionsConfigService) => {
-        const config = configService.getJourneyConfig();
-        const journeyConfig = new TransactionsJourneyConfiguration();
-        journeyConfig.pageSize = config.pageSize;
-        journeyConfig.slimMode = config.slimMode;
-        return journeyConfig;
-      },
-      deps: [TransactionsConfigService],
+export const TRANSACTIONS_PROVIDERS = [
```

Comment:
I'm not sure any of these are needed?  Looks like they're all provided in the journey function above in the new standalone way?  E.g., `withConfig` replaces the need for explicitly providing `TransactionsJourneyConfiguration`, etc.

`TransactionsConfigService` doesn't look like it's used at all, except in the factory functions below.

`TRANSACTIONS_JOURNEY_CONFIG` and `TransactionsJourneyConfiguration` seem to be duplicate providers for the same thing - looks like TRANSACTIONS_JOURNEY_CONFIG is never actually injected anywhere.